services-helpers.md

```diff
@@ -18,6 +18,7 @@ October includes a variety of "helper" PHP functions. Many of these functions ar
 <div class="collection-method-list" markdown="1">
 [array_add](#method-array-add)
 [array_divide](#method-array-divide)
+[array_undot](#method-array-undot)
```

Comment:
Shouldn't this come after array_dot in the menu?

Reply:
Whoops - fixed.